adcm_client/objects.py

```diff
@@ -937,8 +937,19 @@ def _upload(self, bundle_stream) -> Bundle:
         return self.bundle(bundle_id=data['id'])
 
     @allure_step('Upload bundle from "{1}"')
-    def upload_from_fs(self, dirname) -> Bundle:
-        return self._upload(stream.file(dirname))
+    def upload_from_fs(self, dirname, **args) -> Bundle:
+        streams = stream.file(dirname, **args)
+        if len(streams) > 1:
+            raise TooManyArguments('Build more then one bundle from dir')
```

Comment:
I don't like that message. It's better to say what to do.

> upload_bundle_from_fs is not capable with building multiple bundle editions from one dir. Use upload_from_fs_all instead.

Reply:
Fixed.